### PR TITLE
Added bootstrap, devise, and created migrations according to the plan…

### PR DESCRIPTION
**Pull request by Nikkovichert**
…. Had to edit the id columns to include game, user, and piece.

It all works locally for me, so I'm pretty sure it's all in order.

### CODE REVIEW

File: db/migrate/20180409063500_create_games.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateGames < ActiveRecord::Migration[5.0]
+  def change
+    create_table :games do |t|
+      t.integer :id_for_game
+      t.integer :name
```
Comment (ilyakrasnov) [inline db/migrate/20180409063500_create_games.rb]: should be string :)

File: db/migrate/20180409063500_create_games.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      t.string :black_player_id
```
Comment (ilyakrasnov) [inline db/migrate/20180409063500_create_games.rb]: should be integer, as well as `white_player_id`

File: db/migrate/20180409063500_create_games.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateGames < ActiveRecord::Migration[5.0]
+  def change
+    create_table :games do |t|
+      t.integer :id_for_game
```
Comment (ilyakrasnov) [inline db/migrate/20180409063500_create_games.rb]: rails will automatically create an `id` field - no need to for `id_for_game`

File: db/migrate/20180409063544_create_pieces.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.integer :id_for_piece
```
Comment (ilyakrasnov) [inline db/migrate/20180409063544_create_pieces.rb]: same here - no need for `id_for_piece`

**Comment by Nikkovichert** [comment]
ok, sounds good, i'll make the necessary changes, thanks!

File: app/models/user.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :created_games, :class_name => 'Game' 
+  has_many :joined_games, :class_name => 'Game' 
+  has_many :games
```
Comment (ilyakrasnov) [inline app/models/user.rb]: no need for `has_many :games`

File: app/models/user.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :created_games, :class_name => 'Game'
```
Comment (ilyakrasnov) [inline app/models/user.rb]: we'd need to tell rails here how to find the user from the game.. 

lets add sth like: 

```
has_many :created_games, :class_name => 'Game', foreign_key: "owner_id" 
has_many :joined_games, :class_name => 'Game',  foreign_key: "opponent_id"
```

**Comment by ilyakrasnov** [comment]
@Nikkovichert please review the comments.

There seem to be some merge conflicts - which we'll need to resolve before merging. 

Otherwise seems great once resolved 👍

**Pull request merged**